pkg/gadgets/networkpolicy/networkpolicy.go

```diff
@@ -140,7 +140,7 @@ func (a *NetworkPolicyAdvisor) networkPeerKey(e types.KubernetesConnectionEvent)
 	} else if e.RemoteKind == "other" {
 		ret = e.RemoteKind + ":" + e.RemoteOther
 	}
-	return ret + ":" + string(e.Port)
+	return ret + ":" + fmt.Sprint(e.Port)
```

Comment:
```suggestion
	return fmt.Sprintf("%s:%d", ret, e.Port)
```

Does it make sense to use that way now that we're using `fmt.Sprint`?